### DISCUSSION TITLE
howto import downloaded python modules

### DISCUSSION THREAD

**Discussion by pmeyerson** [go-python/cpy3#31] [Usage Discussions]
I'm finding that to import non-system python modules I need to set `PYTHONPATH` to `./venv/lib/python3.7/site-packages` prior to executing my go module, otherwise I get `'package name' not found` on below.

IE python.PyRun_SimpleString(\`import foo
foo.x()
print("it worked")\`)

Is this true for everyone else or is there something weird about my python3.7 setup?
I'd be happy to submit readme update if its not just me.

Running on osx (not M1)

**Comment by christian-korneck** [comment]
@pmeyerson I think this is to be expected. When embedded the Python runtime knows little about the process it runs in (I.e. it doesn't know about the process args unless you tell it).

There are various ways to get module imports to work. I believe this is the idiomatic one:
https://docs.python.org/3/c-api/init.html#c.PySys_SetArgvEx

(I personally tend to just append the path to my modules to `sys.path` via `PyRun_SimpleString`)
